ui/src/utils/groupByTimeSeriesTransform.js

```diff
@@ -2,50 +2,60 @@ import _ from 'lodash'
 import {shiftDate} from 'shared/query/helpers'
 import {map, reduce, forEach, concat, clone} from 'fast.js'
 
-const groupByMap = (results, responseIndex, groupByColumns) => {
+const flattenGroupBySeries = (results, responseIndex, tags) => {
   if (_.isEmpty(results)) {
     return []
   }
+
+  const tagsKeys = _.keys(tags)
+
+  const seriesArray = _.get(results, [0, 'series'])
+  let valuesAcc
+  valuesAcc = []
+
+  seriesArray.forEach(s => {
```

Comment:
I think you can do this with a reduce instead of forEach since you are just accumulating values.  This way you won't need to use let for valuesAcc.